Project-3/Peihong_Xie_Project_3_50375781.md

```diff
@@ -0,0 +1,381 @@
+
+```
+For any question involving the use of Protege, please be sure to import Basic Formal Ontology (https://raw.githubusercontent.com/BFO-ontology/BFO/v2019-08-26/bfo_classes_only.owl) and the Relations Ontology (https://raw.githubusercontent.com/oborel/obo-relations/master/ro.owl)
+```
+
+**[1]** In BFO and RO identify at least one object property for each of a-e that _should have the listed property, but which does not_; argue for your case, using examples. 
+```
+  (a)  Reflexive
+  (b)  Transitive 
+  (c)  Symmetric
+  (d)  Functional
+  (e)  Symmetric and Reflexive
+```
+
+**Answer:**
+
+(a) Object properties **"simultaneous with", "has part", "part of", "overlaps" and so on** should be reflexive, though it is not listed as so. For example, an event can be simultaneous with itself; also, an entity has_part, part_of, or overlaps itself because it is equivalent to itself.
+
+(b) The object property **"determined by"** should be transitive, though it is not listed as so. That is, if A is determined by B, and B is determined by C, then A should be determined by C. For example, a small-size ecological cycle is determined by a biological host, say, a human being, that is, as a system at another level of granularity, is determined by his brain. In this case, we can say that the ecological cycle is determined by the human being's brain because its proper functioning leads the human to make efforts in maintaining that cycle. 
+
+(c) The object property **"biotically interacts with"** should be symmetric, though it is not listed as so. If an organism A biotically interacts with another organism B, then it is clear that B is also biotically interacts with A. 
+
+Similarly, **"participates in a biotic-biotic interaction with", "ecologically co-occurs with", "partially overlap" and so on** should also be symmetric.
+
+(d) The object property **"has skeleton"** should be functional, though it is not listed as so. According to its definition, "has skeleton" is a relation between a segment or subdivision of an organism and the maximal subdivision of material entities that provides structural support for that segment or subdivision. Given that the maximal subdivision of material entities for any organism has only one, this object property is thereby functional.
+
+(e) The object property **"overlaps"** should be both symmetric and reflexive, though it is not listed as so. First, if A overlaps B, then A and B has some common part. So B and A has also some common part, and thus B overlaps A. That is, overlapping is symmetric. Second, an entity can overlap itself because it clearly has the same parts with itself. Therefore, overlapping is also reflexive.
+
+Similarly, **"spatially coextensive with" and "simultaneous with"** is also symmetric and reflexive.
```

Comment:
Same with my previous comments on the extra examples. More likely to get one wrong and doesn't really add anything to your answer.